### PR TITLE
[second issue #2] 이슈 테스트

### PR DESCRIPTION
**Pull request by ldcc-education**
[https://github.com/giseoplee/Chatbot/issues/2](url)
second issue link

### CODE REVIEW

**Comment by giseoplee** [comment]
apply comment

**Comment by ldcc-education** [comment]
apply comment

**Comment by giseoplee** [comment]
request request comment!

File: Controller/KakaoController.js [hunk 1]
```diff
@@ -1,73 +1,72 @@
...
-
-    res.header('Content-Type', 'application/json; charset=utf-8');
-    next();
-});
-
-router.get('/keyboard', (req, res, callback) => {
-
-    res.json({ "type" : "buttons", "buttons" : ["대화하기"] });
+  res.header("Content-Type", "application/json; charset=utf-8");
+  next();
```
Comment (giseoplee) [inline Controller/KakaoController.js:11]: add code

**Pull request merged**